replications/service.go

```diff
@@ -63,6 +64,10 @@ type DurableQueueManager interface {
 	InitializeQueue(replicationID platform.ID, maxQueueSizeBytes int64) error
 	DeleteQueue(replicationID platform.ID) error
 	UpdateMaxQueueSize(replicationID platform.ID, maxQueueSizeBytes int64) error
+	QueuePath() string
+	ReplicationQueues() map[platform.ID]*durablequeue.Queue
```

Comment:
I think we can get away without adding `ReplicationQueues()`.

I don't see where `QueuePath()` is used in this file. If it's used elsewhere it should be added to a different interface (or not added to an interface, if it's only used internally to the queue manager)